satpy/enhancements/colormap.py

```diff
@@ -43,7 +43,9 @@ def lookup(img, **kwargs):
 def _lookup_table(band_data, luts=None, index=-1):
     # NaN/null values will become 0
```

Comment:
Given this comment, it would seem this was supposed to be happening anyway? Or is it just that this comment is referencing the implicit (and warned about) behavior inside `astype` and this is now handled explicitly?